### DISCUSSION TITLE
Document how to use Supabase to provide "serverless"

### DISCUSSION THREAD

**Discussion by fire** [sqlpage/SQLPage#81] [Announcements]
Hi,

I think it's possible to serve SQLpage through supabase.

Can you check and document?

**Comment by lovasoa** [comment]
Hello @fire and welcome to SQLPage ! 

I'm moving this thread to "discussions" since it's not really an issue with SQLPage.

I don't think supabase lets you run custom applications on their infrastructure, but I may be wrong.

But there is a supported serverless deployment for SQLPage: using AWS lambda. When you deploy your SQLPage instance on a lambda, you can still have your database provided by supabase (or any other provider).

Whatever solution you end up choosing, I'd love to receive a small PR from you to the documentation explaining the setup !
